packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js

```diff
@@ -25,6 +25,7 @@ const ContainerGrid = styled.div`
     grid-template-columns: 100%;
     grid-template-rows: auto auto;
     grid-auto-flow: column;
+    grid-gap: 1rem;
```

Comment:
IIRC we're using multiples of `10px` for spacing in the theme?

Reply:
I used rem here since the layout is already using rem, but I can switch to pixels. I'd like to refactor it to use the Grommet Grid here at a later point, so we don't have to manually set this.